src/main/java/com/opentable/versionedconfig/VersionedConfig.java

```diff
@@ -29,21 +28,10 @@
     static final String X_OAUTH_BASIC = "x-oauth-basic";
 
     @Bean
-    public GitProperties defaultVersioningServiceProperties(@Value("${config.repo.remote}") URI remoteRepo,
-                                                            @Value("${config.repo.oauth-token:#{null}}") String token,
-                                                            @Value("${config.repo.username:#{null}}") String username,
-                                                            @Value("${config.repo.password:#{null}}") String password,
-                                                            @Value("${config.repo.local:#{null}}") File localPath,
+    public GitProperties defaultVersioningServiceProperties(@Value("${config.repo.remote}") List<URI> remoteRepo,
```

Comment:
we need to document this change in the log, stating credentials have been moved to uri.